typings/react-native-code-push.d.ts

```diff
@@ -242,13 +249,6 @@ declare namespace CodePush {
     function checkForUpdate(deploymentKey?: string): Promise<RemotePackage>;
 
     /**
-     * Decorates a React Component configuring it to sync for updates with the CodePush server.
-     *
-     * @param options Options used to configure the end-user sync and update experience (e.g. when to check for updates?, show an prompt?, install the update immediately?).
-     */
-    function codePushify(options?: CodePushOptions): Function;
-
-    /**
```

Comment:
`codePushify` is still a property on CodePush in the implementation, we should probably make it consistent one way or another